### PR TITLE
Change rmapy repo used, update Readme

### PR DESCRIPTION
**Pull request by rafaelsgirao**
Hi, first of all thanks for this awesome tool, which I haven't been able to get working yet.

This PR is my first one so please let me know if there's something that needs changing.

It fixes some issues I had so far, points out that the rmapy wiki is not up do date, fixes a typo in Readme and changes the repo used for rmapy : the one that's used now is not really updated anymore and this one is the most up to date.

I also never got this to work anyway, but I'm opening an Issue for that.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,12 +88,13 @@ This script uses hershey stroke fonts to place text in Remarkable lines files wi
...
 Currently, one should initialize their connection to Remarkable Cloud using rmapy directly. You can obtain a one time token from [Remarkable's Mobile Connect](https://my.remarkable.com/connect/mobile) page.
 
-Then follow example as [described here](https://rmapy.readthedocs.io/en/latest/quickstart.html#registering-the-api-client). Eg,
+The official Wiki for rmapy has obsolete syntax, but may still prove useful as examples to follow if you're lost.[You can find it here.](https://rmapy.readthedocs.io/en/latest/quickstart.html#registering-the-api-client). Eg,
 
 ```python
 from rmapy.api import Client
 rmapy = Client()
 rmapy.register_device("<YOUR ONE TIME TOKEN>")
+rmapy.renew_token()
```
Comment (bsdz) [inline README.md:97]: This shouldn't be necessary as it's already renewed in rmcloud.py https://github.com/bsdz/remarkable-layers/blob/f96a73a4c6da2ce93ec42f21e7964a77a910af63/rmlines/rmcloud.py#L18

File: requirements.txt [hunk 2]
```diff
@@ -8,7 +8,7 @@ pyaml==19.4.1
 -e git+https://github.com/flupke/pypotrace.git@76c76be2458eb2b56fcbd3bec79b1b4077e35d9e#egg=pypotrace
 pyyaml==5.3.1
 requests==2.24.0
--e git+https://github.com/bsdz/rmapy.git@d53983e68021402f6b1dc8d736633a9c53f903b2#egg=rmapy
+-e git+https://github.com/j6k4m8/rmapy.git@d53983e68021402f6b1dc8d736633a9c53f903b2#egg=rmapy
```
Comment (bsdz) [inline requirements.txt]: Why switch from my fork? My fork has been tested and works at least for myself and a few others.

File: README.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ Convert simple SVG file into RM Lines format. Can only contain paths with simple
 
 ```python
 p = Path("./my_simple.svg")
-rm0 = RMLines.from_svg(f.open("rb")
+rm0 = RMLines.from_svg(f.open("rb"))
```
Comment (bsdz) [inline README.md:61]: Good catch!

**Review by bsdz — CHANGES_REQUESTED** [review]
Thanks for your PR. I can see some improvements although there are some changes I'm not quite sure I understand.

File: requirements.txt (hunk 2 above)
Comment (rafaelsgirao) [reply requirements.txt]: Right, this isn't necessary - I thought it was a problem with the fork when I kept getting stuck at Not Authenticated Exceptions and happened to fix it when I did renew_token(). Just tested with your fork and it works fine, sorry about that.

File: README.md (hunk 1 above)
Comment (rafaelsgirao) [reply README.md:97]: Actually, I don't think this ever gets executed - for me atleast, the exception right above is raised and execution is stopped, so rmapy.renew_token() never gets called.

**Pull request closed without merging**